modules/dr-restoring-cluster-state.adoc

```diff
@@ -287,6 +287,11 @@ If the status is `Pending`, or the output lists more than one running etcd pod,
 
 . Delete and recreate other non-recovery, control plane machines, one by one. After these machines are recreated, a new revision is forced and etcd scales up automatically.
 +
+[WARNING]
+====
+For baremetal IPI deployments control plane machines are not recreated. Follow the process as described in "Replacing a bare-metal control plane node". For more information, see the additional resources section.
```

Comment:
My understanding is that the recommendation is not to use the acronyms "IPI" or "UPI" in the product documentation. You could update "For baremetal IPI deployments" to avoid using "IPI". You could also split "baremetal" into two words:

> For bare metal installations on installer-provisioned infrastructure, ...